src/widgetastic_patternfly4/clipboardcopy.py

```diff
@@ -39,3 +35,22 @@ def fill(self, value):
 
     def copy(self):
         self.button.click()
+
+
+class ClipboardCopy(BaseClipboardCopy, Widget):
+    ROOT = ParametrizedLocator("{@locator}")
```

Comment:
Defining this along with the `__init__` implementation of `DEFAULT_LOCATOR` is not a pattern I'd recommend.  I think the base class can be cleaned up to just set ROOT of `BaseClipboardCopy` to what you have here for DEFAULT_LOCATOR.